proofread: mdtest.md (#1571)
juicedata/juicefs: docs/en/benchmark/mdtest.md

# Benchmark with mdtest

:::tip

> JuiceFS v1.0+ has Trash enabled by default. The benchmark creates and deletes temporary files in the file system, and eventually dumps these files into a directory named `.trash`. To avoid storage space being occupied by `.trash`, you can run command `juicefs config META-URL --trash-days 0` to perform benchmark with Trash disabled. See [trash](../security/trash.md) for details.

Reviewer: ```suggestion
Trash is enabled in JuiceFS v1.0+ by default. As a result, temporary files are created and deleted in the file system during the benchmark, and these files will be eventually dumped into a directory named `.trash`. To avoid storage space being occupied by `.trash`, you can run command `juicefs config META-URL --trash-days 0` to disable Trash before benchmark. See [trash](../security/trash.md) for details.
```
稍微改了一下语序尝试着把句子变得更通畅一些，你看看行不行哈

Author: 没问题 👌

:::

## Testing Approach

Perform a metadata test on JuiceFS, [EFS](https://aws.amazon.com/efs) and [S3FS](https://github.com/s3fs-fuse/s3fs-fuse) with [mdtest](https://github.com/hpc/ior).

## Testing Tool

The following tests are performed with `mdtest` 3.4.

> Tune arguments of `mdtest` to ensure that the command will finish in 5 minutes.

Reviewer: ```suggestion
The arguments of `mdtest` are tuned to ensure that the command will finish within 5 minutes.
```
感觉这里用倒装似乎更好一些？

## Testing Environment

> The following tests are all performed using `mdtest` on a c5.large EC2 instances (2 CPU, 4G RAM) with Ubuntu 18.04 LTS (Kernel 5.4.0) operating system. The Redis (version 4.0.9) which JuiceFS uses runs on a c5.large EC2 instance in the same available zone to store metadata.

Reviewer: ```suggestion
All the following tests are performed using `mdtest` on a c5.large EC2 instances (2 CPU, 4G RAM) with Ubuntu 18.04 LTS (Kernel 5.4.0) operating system. The Redis (version 4.0.9) which JuiceFS uses runs on a c5.large EC2 instance in the same available zone to store metadata.
```
有个小问题：这里的<区域>的专业术语是<zone>嘛？还是说可以用<region/area>？我有点不太明白这里的区域是指什么呢？:P

Author: 购买亚马逊云服务器的时候可以选择服务器所在地区的，比如「北美」「亚洲」「俄勒冈」「东京」这样。官方用的是 region，我觉得 available zone 也问题不大
